suite-common/wallet-utils/src/reviewTransactionUtils.ts

```diff
@@ -382,6 +386,22 @@ const constructNewFlow = ({
         });
     }
 
+    if (isEvmApprovalTx && evmApprovalTxData && networkType === 'ethereum') {
+        outputs.push({
+            type: 'contract',
+            value: EVM_SPENDER_LABELS[evmApprovalTxData[1].spender] || evmApprovalTxData[1].spender,
+        });
+
+        if (precomposedTx.token) {
+            outputs.push({
+                type: 'approve_data',
+                value: evmApprovalTxData[1].amount,
+                value2: capitalizeFirstLetter(precomposedForm.network?.name || ''),
```

Comment:
`value2` can be set to `undefined`, wouldn't it be better to do:
```suggestion
                value2: precomposedForm.network?.name ? capitalizeFirstLetter(precomposedForm.network?.name) : undefined,
```
so that it cannot happen that the `value2` is shown but with empty string?